### PR TITLE
SUS-1811: Get rid of a needless wait

### PR DESCRIPTION
**Pull request by TK-999**
Removed a redundant wait. This isn't used in reply tests, so should not be necessary here, and seems to be a major cause for random Wall test failures.

https://wikia-inc.atlassian.net/browse/SUS-1811

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [MessageWall.java#L43](https://github.com/Wikia/selenium-tests/blob/c7488a23e94efac5985077ab96643383d4dc4a7e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java#L43): Make this final field static too. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1170)
1. MAJOR [MessageWall.java#L45](https://github.com/Wikia/selenium-tests/blob/c7488a23e94efac5985077ab96643383d4dc4a7e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java#L45): Make this final field static too. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1170)
1. MAJOR [MessageWall.java#L46](https://github.com/Wikia/selenium-tests/blob/c7488a23e94efac5985077ab96643383d4dc4a7e/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java#L46): Make this final field static too. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1170)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [MessageWall.java#L43](https://github.com/Wikia/selenium-tests/blob/9f070b38bf62b87aed08b597602a848263653a6a/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java#L43): Make this final field static too. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1170)
1. MAJOR [MessageWall.java#L45](https://github.com/Wikia/selenium-tests/blob/9f070b38bf62b87aed08b597602a848263653a6a/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java#L45): Make this final field static too. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1170)
1. MAJOR [MessageWall.java#L46](https://github.com/Wikia/selenium-tests/blob/9f070b38bf62b87aed08b597602a848263653a6a/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java#L46): Make this final field static too. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1170)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 1]
```diff
@@ -21,31 +20,31 @@
...
-  private final By replyBodyBy = By.cssSelector(".replyBody");
-  private final String newMessageMenu =
+  final static By replyButtonBy = By.cssSelector(".replyButton");
+  private final static By messageTitleBy = By.cssSelector(".msg-title");
+  private final static By messageBodyBy = By.cssSelector(".msg-body");
+  private final static By imageBy = By.cssSelector(".thumbimage");
+  private final static By messageTextBoldBy = By.cssSelector("b");
+  private final static By messageTextItalicBy = By.cssSelector("i");
+  private final static By messageTextBy = By.cssSelector(".msg-body *");
+  private final static By messageLinkBy = By.cssSelector("a");
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 2]
```diff
@@ -21,31 +20,31 @@
...
+  private final static By messageUserNameBy = By.cssSelector(".edited-by > a:nth-child(1)");
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 3]
```diff
@@ -21,31 +20,31 @@
...
+  private final static By moreButtonBy = By.cssSelector(".wikia-menu-button.secondary.combined");
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 4]
```diff
@@ -21,31 +20,31 @@
...
+  private final static By editButtonBy = By.cssSelector(".edit-message");
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 5]
```diff
@@ -21,31 +20,31 @@
...
+  private final static By removeButtonBy = By.cssSelector(".remove-message");
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 6]
```diff
@@ -21,31 +20,31 @@
...
+  private final static By reopenButtonBy = By.cssSelector(".reopen-thread");
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 7]
```diff
@@ -21,31 +20,31 @@
...
+  private final static By quoteButtonBy = By.cssSelector(".quote-button.secondary");
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 8]
```diff
@@ -21,31 +20,31 @@
...
+  private final static By quoteMessageBy = By.cssSelector(".replies p");
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 9]
```diff
@@ -21,31 +20,31 @@
...
+  private final static By saveChangesButtonBy = By.cssSelector(".save-edit");
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 10]
```diff
@@ -21,31 +20,31 @@
...
+  private final static By closeThreadInfobox = By.cssSelector(".deleteorremove-bubble > .message");
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 11]
```diff
@@ -21,31 +20,31 @@
...
+  private final static By replyBodyBy = By.cssSelector(".replyBody");
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 12]
```diff
@@ -21,31 +20,31 @@
...
+  private final static String newMessageMenu =
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MAJOR Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS00115)
MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 13]
```diff
@@ -21,31 +20,31 @@
...
       ".comments li.SpeechBubble.message.message-main:nth-child(1) .buttons";
-  private final String firstMessageMenu = ".comments li:nth-child(1) .buttons ";
-  private final String closeButtonString = ".close-thread";
-  private final By closeButtonBy = By.cssSelector(firstMessageMenu + closeButtonString);
+  private final static String firstMessageMenu = ".comments li:nth-child(1) .buttons ";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MAJOR Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS00115)
MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 14]
```diff
@@ -21,31 +20,31 @@
...
+  private final static String closeButtonString = ".close-thread";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MAJOR Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS00115)
MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 15]
```diff
@@ -21,31 +20,31 @@
...
+  private final static By closeButtonBy = By.cssSelector(firstMessageMenu + closeButtonString);
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 16]
```diff
@@ -21,31 +20,31 @@
 
 public class MessageWall extends WikiBasePageObject {
 
-  final By firstMessageWrapperBy =
+  final static By firstMessageWrapperBy =
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 17]
```diff
@@ -21,31 +20,31 @@
...
-  private final By editButtonBy = By.cssSelector(".edit-message");
-  private final By removeButtonBy = By.cssSelector(".remove-message");
-  private final By reopenButtonBy = By.cssSelector(".reopen-thread");
-  private final By quoteButtonBy = By.cssSelector(".quote-button.secondary");
-  private final By quoteMessageBy = By.cssSelector(".replies p");
-  private final By saveChangesButtonBy = By.cssSelector(".save-edit");
-  private final By closeThreadInfobox = By.cssSelector(".deleteorremove-bubble > .message");
-  private final By replyBodyBy = By.cssSelector(".replyBody");
-  private final String newMessageMenu =
+  final static By replyButtonBy = By.cssSelector(".replyButton");
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 18]
```diff
@@ -21,31 +20,31 @@
...
+  private final static By messageTitleBy = By.cssSelector(".msg-title");
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 19]
```diff
@@ -21,31 +20,31 @@
...
+  private final static By messageBodyBy = By.cssSelector(".msg-body");
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 20]
```diff
@@ -21,31 +20,31 @@
...
+  private final static By imageBy = By.cssSelector(".thumbimage");
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 21]
```diff
@@ -21,31 +20,31 @@
...
+  private final static By messageTextBoldBy = By.cssSelector("b");
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 22]
```diff
@@ -21,31 +20,31 @@
...
+  private final static By messageTextItalicBy = By.cssSelector("i");
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 23]
```diff
@@ -21,31 +20,31 @@
...
+  private final static By messageTextBy = By.cssSelector(".msg-body *");
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 26 issues
* MAJOR 3 major
* MINOR 23 minor

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 24]
```diff
@@ -21,31 +20,31 @@
...
+  private static final By moreButtonBy = By.cssSelector(".wikia-menu-button.secondary.combined");
+  private static final By editButtonBy = By.cssSelector(".edit-message");
+  private static final By removeButtonBy = By.cssSelector(".remove-message");
+  private static final By reopenButtonBy = By.cssSelector(".reopen-thread");
+  private static final By quoteButtonBy = By.cssSelector(".quote-button.secondary");
+  private static final By quoteMessageBy = By.cssSelector(".replies p");
+  private static final By saveChangesButtonBy = By.cssSelector(".save-edit");
+  private static final By closeThreadInfobox = By.cssSelector(".deleteorremove-bubble > .message");
+  private static final By replyBodyBy = By.cssSelector(".replyBody");
+  private static final String newMessageMenu =
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MAJOR Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS00115)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 25]
```diff
@@ -21,31 +20,31 @@
...
       ".comments li.SpeechBubble.message.message-main:nth-child(1) .buttons";
-  private final String firstMessageMenu = ".comments li:nth-child(1) .buttons ";
-  private final String closeButtonString = ".close-thread";
-  private final By closeButtonBy = By.cssSelector(firstMessageMenu + closeButtonString);
+  private static final String firstMessageMenu = ".comments li:nth-child(1) .buttons ";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MAJOR Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS00115)

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java [hunk 26]
```diff
@@ -21,31 +20,31 @@
...
+  private static final String closeButtonString = ".close-thread";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: MAJOR Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS00115)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* MAJOR 3 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java (hunk 26 above)
Comment (TK-999) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/messagewall/MessageWall.java]: spierdalaj

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 3 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by TK-999** [comment]
go away

**Pull request closed without merging**